wiki/People/The_Team/Beatmap_Nominators/id.md

```diff
@@ -2,45 +2,43 @@
+Full Beatmap Nominator (Full BN) merupakan para anggota Beatmap Nominator yang memiliki kuasa penuh akan seluruh hak BN. Di samping mengecek dan menominasikan beatmap, para anggota Full BN juga dapat mendiskualifikasi beatmap Qualified yang dipandang bermasalah. Mereka juga dapat  berpartisipasi dalam proses evaluasi calon anggota Beatmap Nominator baru apabila mereka tertarik untuk bergabung ke Nomination Assessment Team ke depannya.
 
-Kira-kira setiap tiga bulan, perilaku dan aktivitas/kualitas nominasi dari full Beatmap Nominator dinilai oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team). Jika nominasi mereka sering di reset karena alasan yang penting atau perilaku mereka yang tidak dapat diterima, mereka mungkin akan dipindahkan ke probationary Beatmap Nominators.
+Setiap kurang lebihnya tiga bulan sekali, perilaku dan aktivitas/kualitas nominasi para anggota Full Beatmap Nominator akan dinilai oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team). Apabila nominasi mereka sering dianulir karena alasan yang fatal atau mereka melakukan sesuatu yang tidak dapat diterima, status keanggotaan mereka dapat diturunkan menjadi Probationary Beatmap Nominator.
 
-Full Beatmap Nominator dengan aktivitas nominasi dibawah minimum tidak akan ditempatkan dalam masa percobaan, namum mereka akan akan dikeluarkan dari kelompok jika aktivitas mereka tidak meningkat.
+Para anggota Full Beatmap Nominator yang tidak memenuhi persyaratan jumlah nominasi minimum per bulannya tidak akan langsung ditempatkan sebagai Probationary Beatmap Nominator, namun mereka beresiko untuk dikeluarkan dari Beatmap Nominator apabila aktivitas mereka tidak kunjung meningkat.
 
 ### Probationary Beatmap Nominator
 
-Masa percobaan digunakan untuk memantau Beatmap Nominator baru lebih dekat. Peran utama dari probationary Beatmap Nominators adalah membangun dasar sebagai seorang Beatmap Nominator dengan mempromosikan beatmap secara eksklusif pada proses ranking. Setidaknya satu full nominator harus dilibatkan dalam proses ranking untuk setiap beatmap, jadi anggota probationary tidak dapat memberikan nominasi akhir terhadap beatmap yang mendapatkan nominasi dari anggota probationary lainnya.
+Probationary Beatmap Nominator (Probationary BN) merupakan para anggota Beatmap Nominator yang berada dalam masa percobaan dan tidak memiliki kuasa penuh akan seluruh hak BN. Tujuan utama dari posisi ini adalah untuk memupuk nilai dasar Beatmap Nominator kepada para anggotanya, di mana satu-satunya tugas dan kuasa yang mereka miliki adalah untuk menominasikan beatmap. Di samping itu, berhubung proses ranking harus melibatkan setidaknya satu Full Beatmap Nominator, para anggota Probationary Beatmap Nominator tidak dapat memberikan nominasi kedua pada beatmap yang sebelumnya telah dinominasikan oleh anggota Probationary Beatmap Nominator lainnya.
```

Comment:
```suggestion
Probationary Beatmap Nominator (Probationary BN) merupakan para anggota Beatmap Nominator yang berada dalam masa percobaan dan tidak memiliki kuasa penuh akan seluruh hak BN. Tujuan utama dari posisi ini adalah untuk menstabilkan nilai dasar Beatmap Nominator kepada para anggotanya, di mana satu-satunya tugas dan kuasa yang mereka miliki adalah untuk menominasikan beatmap. Di samping itu, berhubung proses ranking harus melibatkan setidaknya satu Full Beatmap Nominator, para anggota Probationary Beatmap Nominator tidak dapat memberikan nominasi kedua pada beatmap yang sebelumnya telah dinominasikan oleh anggota Probationary Beatmap Nominator lainnya.
```

Reply:
"memupuk" is not a word to use in this case 🎋

Reply:
changed into "mengukuhkan" instead, should be better now i think